src/pages/SchedulePage.js

```diff
@@ -1,119 +1,280 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import PropTypes from "prop-types";
 import { connect } from "react-redux";
+import { dayToStr } from "../constants";
+import WishlistCourse from "../components/WishlistCourse";
+import Calendar from "../components/Calendar";
+import ScheduleCourse from "../components/ScheduleCourse";
+
 import styled from "styled-components";
+import SnackBar from "@material-ui/core/Snackbar";
+import { grey } from "@material-ui/core/colors";
+
+import * as wishlistActions from "../redux/modules/wishlist";
+import * as courseActions from "../redux/modules/courseSelect";
+
+function SchedulePage({
+  year,
+  semester,
+  wishlist,
+  wishlistCourse,
+  scheduled,
+  toggleCourseSelect,
+  clearSchedule,
+}) {
+  const [schedule, setSchedule] = useState({});
+  const [checkboxes, setCheckboxes] = useState(
+    JSON.parse(localStorage.getItem(`${year}-${semester}-checkbox-state`)) || {}
+  );
+  const [Toast, setToast] = useState({
+    open: false,
+    message: "",
+    horizontal: "center",
+    vertical: "top",
+  });
+  const { open, message, horizontal, vertical } = Toast;
+
+  useEffect(() => {
```

Comment:
Add some new-line spacing in this useEffect to separate different parts of the code so it isn't one long chunk.